src/code/translations/english.json

```diff
@@ -237,7 +237,7 @@
   "OP_BUTTON": "Operation",
   "OP_SETTINGS_BUTTON": "Op ⚙",
   "OP_SETTINGS_TITLE": "Operation Settings",
-  "OPTION": "option",
```

Comment:
in fact this string looks never used

Reply:
My understanding is a pass to look for unused keys is coming soon.